utils/admins.go

```diff
@@ -0,0 +1,6 @@
+package utils
+
+var ADMINS = map[string]bool{
```

Comment:
Might be a good opportunity to rename this from `ADMINS` to `Admins` to abide by the golang style guide

Reply:
done